scripts/install.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+get_architecture() {
+  local _ostype _cputype _bitness _arch _clibtype
+  _ostype="$(uname -s)"
+  _cputype="$(uname -m)"
+  _clibtype="gnu"
+
+  if [ "$_ostype" = Linux ]; then
+    if ldd --_requested_version 2>&1 | grep -q 'musl'; then
+      _clibtype="musl"
+    fi
+  fi
```

Comment:
nitpick:

this can probably be rewritten to avoid nested ifs. I haven't tested this, but something like:
```
if [ "$_ostype" = Linux ] && ldd --_requested_version 2>&1 | grep -q 'musl'; then
  _clibtype="musl"
fi
```

should probably work.